ZenPacks/zenoss/Microsoft/Windows/zenpack.yaml

```diff
@@ -94,7 +94,7 @@ zProperties:
     label: 'Regex expressions to model services with generic class'
     description: 'List of regular expressions for Windows Service classes to model services with generic Windows Service class.'
     type: lines
-    default: ['BcastDVRUserService', 'BluetoothUserService', 'CaptureService', 'CDPUserSvc', 'DevicePickerUserSvc', 'DevicesFlowUserSvc', 'MessagingService', 'OneSyncSvc', 'PimIndexMaintenanceSvc', 'PrintWorkflowUserSvc', 'UnistoreSvc', 'UserDataSvc', 'WpnUserService']
+    default: ['BcastDVRUserService', 'BluetoothUserService', 'CDPUserSvcCaptureService', 'ConsentUxUserSvc', 'CredentialEnrollmentManagerUserSvc', 'DevicePickerUserSvc', 'DevicesFlowUserSvc', 'MessagingService', 'OneSyncSvc', 'PimIndexMaintenanceSvc', 'PrintWorkflowUserSvc', 'UdkUserSvc', 'UnistoreSvc', 'UserDataSvc', 'WpnUserService', 'cbdhsvc']
```

Comment:
'CDPUserSvcCaptureService' is a typo. Need to split this service into 'CDPUserSvc' and 'CaptureService'.

Reply:
Thanks for catching this. I updated the branch with the typo corrected.